packages/http/src/validator/validators/body.ts

```diff
@@ -16,7 +41,7 @@ export class HttpBodyValidator implements IHttpValidator<any, IMediaTypeContent>
     }
 
     if (!content.schema) {
-      return [];
+      return checkExamples(target, mediaType);
     }
```

Comment:
I’m not sure I understand this bit. Why are we changing how things work when no examples are found?

Reply:
@philsturgeon, I'm not sure I understand. The above says: `if there is no schema in the content, then please check *target*, which is response body, which is a user provided example`. Or at least, this is my understanding. And also, as I understand, there was no validation on outputs, only inputs were validated. Output validations were always passed as a `Right` (`Either`).

Reply:
I think @lag-of-death is right, although this is probably not the correct PR to make this happen.

If I understand correctly — the idea is that the example specified by the user shall be validated before being sent to the client.

For instance

```yaml
example:
  application/json: apodkosakdospakd
  application/xml: aisjaiw1023
```

These are probably *not* valid; we're still returning them and send them with their respective headers. I think @lag-of-death was trying to fix this by validating the examples; I think this is something that Spectral should do.

@lag-of-death Did I understand your intention?
@philsturgeon Does it clarify? If so, do you think this is something that Prism should do?

In any case maybe we could create a separate issue to work on this instead of melting the changes together.

Reply:
> I think @lag-of-death is right, although this is probably not the correct PR to make this happen.

That explains it! I was trying to work out how it was related to the feature we are implementing, and its not. Okie doke! 

@lag-of-death could you make a story outlining what is going wrong, what this aims to address, the whole thing, then we can decide if we'd like to work on it later.

Reply:
not so lengthy, but I think to the point issue created: https://github.com/stoplightio/prism/issues/495 LMK if there's still some confusion/anything :+1:
cc @XVincentX, @philsturgeon